javascript/ecmascript-2015/generators-proxy/passing-values-to-generators.md

```diff
@@ -24,58 +18,58 @@ links:
 ---
 ## Content
 
-Generators next method also accepts a parameter.
+Generator's `next` method also accepts a parameter.
 
-When you pass in a parameter it is assigned to the result of the **yield** statement (if you don’t pass anything in this is undefined).
+When you pass in a parameter it is assigned to the result of the `yield` statement (if you don’t pass anything in it is `undefined`).
 
 We could for example use this functionality to reset our number sequence when 0 is passed in:
 
 ```javascript
-function* getGenerator(){
- var currentNumber = 1;
+function* getGenerator() {
+  let currentNumber = 1;
 
- while(currentNumber < 6){
-  var input = yield currentNumber++;
-  if (input==0) currentNumber = 1;
- }
+  while (currentNumber < 6) {
+    let input = yield currentNumber++;
+    if (input == 0) currentNumber = 1;
+  }
 }
 
 var g = getGenerator();
 
-g.next().value //1
-g.next().value //2
-g.next().value //3
-g.next(0).value //1
+g.next().value; // 1
+g.next().value; // 2
+g.next().value; // 3
+g.next(0).value; // 1
 ```
 
 There is however also a gotcha to watch out for when passing values to generators.
```

Comment:
```suggestion
There is also a gotcha to watch out for when passing values to generators.
```

Not sure if "however" is needed here

Reply:
how about this?
```suggestion
There is, however, also a gotcha to watch out for when passing values to generators.
```

Reply:
I think we should use either "however" or "also", but not both